src/pages/Validators/Table.tsx

```diff
@@ -140,6 +147,11 @@ export function ValidatorsTable({
   validators,
   columns = DEFAULT_COLUMNS,
 }: ValidatorsTableProps) {
+  const validatorsInOrder = validators.sort(
```

Comment:
It's not a good idea for a component to mutate its props. You should create a copy of the validators array and sort the copy.